modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -450,8 +450,10 @@ enum SolvePnPMethod {
     SOLVEPNP_ITERATIVE   = 0,
     SOLVEPNP_EPNP        = 1, //!< EPnP: Efficient Perspective-n-Point Camera Pose Estimation @cite lepetit2009epnp
     SOLVEPNP_P3P         = 2, //!< Complete Solution Classification for the Perspective-Three-Point Problem @cite gao2003complete
-    SOLVEPNP_DLS         = 3, //!< A Direct Least-Squares (DLS) Method for PnP  @cite hesch2011direct
-    SOLVEPNP_UPNP        = 4, //!< Exhaustive Linearization for Robust Camera Pose and Focal Length Estimation @cite penate2013exhaustive
+    SOLVEPNP_DLS         = 3, //!< <b> Broken implementation. Using this flag will fallback to EPnP. </b>
+                              //!< <del> A Direct Least-Squares (DLS) Method for PnP @cite hesch2011direct </del>
```

Comment:
> `calib3d/include/opencv2/calib3d.hpp:454: warning: Unsupported xml/html tag <del> found`

http://pullrequest.opencv.org/buildbot/export/pr/18171/docs/d9/d0c/group__calib3d.html#ga357634492a94efe8858d0ce1509da869

Can we use `@note` here instead?

Related SO question: https://stackoverflow.com/questions/27827966/how-can-i-use-strikethrough-in-the-doxygens-markdown

Reply:
I have replaced `<del>` with `~~`. Let see if it works.

Reply:
Looks like strikethrough is not correctly rendered with the version of Doxygen on the CI. I will update to remove it.